python/example01.py

```diff
@@ -0,0 +1,149 @@
+import pycurl
+import os
+import sys
+import json
+import io
+
+AK_SIZE = 20
+SK_SIZE = 40
+
+class Tag:
+    def __init__(self, resources_id, key, value, resources_type):
+        self.resources_id = resources_id
+        self.key = key
+        self.value = value
+        self.resources_type = resources_type
+
+def setup_curl(url, json_data, ak_sk):
+    curl = pycurl.Curl()
+    headers = ["Content-Type: application/json"]
+    curl.setopt(pycurl.HTTPHEADER, headers)
+    curl.setopt(pycurl.URL, url)
+    curl.setopt(pycurl.POSTFIELDS, json_data)
+    curl.setopt(pycurl.AWS_SIGV4, 'osc')
+    curl.setopt(pycurl.USERPWD, ak_sk)
+
+    response_data = []
+
+    def write_data(data):
+        response_data.append(data.decode('utf-8'))
+
+    try:
+        curl.setopt(pycurl.WRITEFUNCTION, write_data)
+        curl.perform()
+    except pycurl.error as e:
+        print("Pycurl error:", e)
+        return None
+
+    curl.close()
+    response_str = "".join(response_data)
+
+    return response_str
+
+def create_tag(tag, ak_sk):
+    data = {"ResourceIds": [tag.resources_id],"Tags": [{"Key": tag.key, "Value": tag.value}]}
+    json_data = json.dumps(data)
+    
+    # Setting the URL for create
+    url = "https://api.eu-west-2.outscale.com/api/v1/CreateTags"
+    
+    if setup_curl(url, json_data, ak_sk):
```

Comment:
you should write the full result of setup_curl, because it could be useful for debug.